src/background/browser-adapters/runtime-adapter.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type RuntimeAdapter = {
+    getRuntimeId(): string;
+    getRuntimeLastError(): chrome.runtime.LastError;
```

Comment:
Similar to above comment for `getRuntimeLastError`; for our purposes, a call to `getRuntimeLastError()` is only meaningful in the context of a coupled implementation of `getSelectedTabInWindow`, so I don't think it makes sense to decouple their interfaces.